addons/point_of_sale/tools/posbox/configuration/posbox_update.sh

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env bash
 
 sudo mount -o remount,rw /
-sudo git --work-tree=/home/pi/odoo/ --git-dir=/home/pi/odoo/.git pull
+
+# As we got an error of diverged branches in the case of git pull,
+# we fetch and checkout the remote branch, delete the local branch and
+# set the remote branch to be the new original one
+sudo git --work-tree=/home/pi/odoo/ --git-dir=/home/pi/odoo/.git fetch --depth=1
```

Comment:
why do we use sudo? aren't we with user `pi`?
```
$ git fetch origin/12.0 --depth=1
$ git reset origin/12.0 --hard
```
should be enough

Reply:
Then why was it sudo before?  You could easily test by trying to do this on the raspberry with user pi.

Reply:
I tried with the user pi and it works, hence my question, I don't know why we put sudo before

Reply:
just need to be sure the script is itself not called with sudo then.

Reply:
in case the script is ran with sudo best practice would be to run the /home/pi parts with `sudo -upi`